### ISSUE TITLE
Release failing

### ISSUE THREAD

**Issue by ajeje93** [nekocodeX/GitHub-Action-Ahk2Exe#5]
Hi,

I'm trying to use this action to compile my AHK script but I am having issues doing it. In particular, it seems that the file is not generated by the action in my case so the release step fails.

Can you help me understand what am I doing wrong?

My action: https://github.com/ajeje93/ahk-italian-shortcuts/blob/master/.github/workflows/release.yml

The action logs:

![2022-01-05 13_02_29-Create release yml · ajeje93_ahk-italian-shortcuts@d69e80c](https://user-images.githubusercontent.com/11637841/148214629-daff36a4-9f74-4221-a25f-0b2f03715962.png)

**Comment by nekocodeX** [comment]
Hi, @ajeje93!
(Sorry for the late reply.)

I just released a new version ([`v1.1.0`](https://github.com/marketplace/actions/ahk2exe?version=v1.1.0)).
I forked your repository and ran the action and it worked until the release.
Could you please try the new version once?
```yaml
...
      - name: Ahk2Exe
        uses: nekocodeX/GitHub-Action-Ahk2Exe@v1.1.0
        with:
          in: shortcuts.ahk
...
```
I hope this will be successful.

**Comment by ajeje93** [comment]
Hi @nekocodeX,
it worked like a charm! Thank you very much!
